communication/src/main/java/PNCInterface.java

```diff
@@ -0,0 +1,31 @@
+import org.jboss.da.communcation.pnc.entity.BuildConfiguration;
+import org.jboss.da.communcation.pnc.entity.BuildConfigurationSet;
+import org.jboss.da.communcation.pnc.entity.Product;
+import org.jboss.da.communcation.pnc.entity.Project;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+public interface PNCInterface {
```

Comment:
In short-term this is OK, but I would consider small refactoring in Orchestrator and providing an artifact with the contract.